src/component/Game.tsx

```diff
@@ -10,7 +10,22 @@ export const Game = () => {
   const [start, setStart] = useState<boolean>(false);
   const [incorrectLetters, setIncorrectLetters] = useState<number>(0);
 
-  const endGame = () => (currentWordIndex < words.length ? false : true);
+  const endGame = () => {
+    if (finalLetter()) {
+      setCurrentWordIndex(currentWordIndex + 1);
+      return true;
+    } else {
+      return currentWordIndex < words.length ? false : true;
+    }
+  };
+  const finalLetter = () => {
```

Comment:
`finalLetter` isn't a great function name as it doesn't imply that we are returning a boolean. It might read better if it was something like `onFinalLetter` or `isFinalLetter`.